internal/bundle/validate_test.go

```diff
@@ -0,0 +1,60 @@
+package bundle
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/cli/libs/dyn/convert"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccBundleValidate(t *testing.T) {
+	testutil.GetEnvOrSkipTest(t, "CLOUD_ENV")
+
+	tmpDir := t.TempDir()
+	testutil.TouchWithContent(t,
```

Comment:
It's not touching a file when you write to it :-) -- this can be a `testutil.WriteFile`.